wp/lib/bap_wp/src/bil_to_bir.ml

```diff
@@ -137,3 +137,6 @@ let bil_to_sub (stmts : Bil.t) : sub term =
   let res_sub = Term.remove blk_t res_sub head_tid in
   let res_sub = Term.prepend blk_t res_sub hd_blk in
   res_sub
+
+let call (call_sub : sub term ) (arch : Bil.typ) : Bil.Types.stmt  =
```

Comment:
I'd call that argument `typ` or `type`. Obviously it does depend (somehow) on the architecture, but it's just the type of the call.

Reply:
Good idea. Changing the name to `typ` will also make it more consistent with the comments I've made.